Jupyter Notebook/src/CartoCosmos.py

```diff
@@ -0,0 +1,283 @@
+from ipyleaflet import *
+import ipywidgets as widgets
+import math as Math
+import json
+import geojson
+import shapely.geometry as geo
+import shapely.wkt
+
+class Planetary_Maps:
```

Comment:
Just being kinda picky about the module names, but it's more common to not capitalize class names and just use underscores for python.